src/View/Helper/LessHelper.php

```diff
@@ -211,8 +213,14 @@ protected function compile(array $input, $cache, array $options = [], array $mod
         $parse = $this->prepareInputFilesForParsing($input);
 
         if ($cache) {
-            $options += ['cache_dir' => $this->cssPath];
-            return \Less_Cache::Get($parse, $options, $modifyVars);
+        	if( !isset($options['cache_dir'])){
+        		$options['cache_dir'] =$this->cssPath;
+        	} else if( substr( $options['cache_dir'], 0, 1 ) =='/'){
+       			$options['cache_dir']  = WWW_ROOT . $options['cache_dir'];
+       		} else {
+       			$options['cache_dir']  = trim($this->cssPath, '/') . $options['cache_dir'];
+	       	}
+        	return \Less_Cache::Get($parse, $options, $modifyVars);
```

Comment:
There are spaces + tabs in many of these lines. Also, the spaces of the `if` are not properly set and you should avoid using `else if`. Instead use `elseif`.

It's also recommendable not to use `else`. Better define the variable before as a fallback.